web/stripe-template/api/routes/webhook/POST-product-price.js

```diff
@@ -0,0 +1,196 @@
+import { RouteContext } from "gadget-server";
+import { stripe, getStripeWebhookEvent } from "../../stripe";
+import { destructure, objKeyConvert } from "../../utils";
+
+// Change all to use background actions
+
+/**
+ * Route handler for POST webhook/product-price
+ *
+ * @param { RouteContext } route context - see: https://docs.gadget.dev/guides/http-routes/route-configuration#route-context
+ *
+ */
+export default async function route({ request, reply, api, logger }) {
+  let event = request.body;
+
+  try {
+    event = getStripeWebhookEvent({
+      request,
+      endpointSecret: process.env.STRIPE_PRODUCT_PRODUCT_PRICE_WEBHOOK_SECRET,
```

Comment:
The name of the env var was updated
```suggestion
      endpointSecret: process.env.STRIPE_PRODUCT_PRICE_WEBHOOK_SECRET,
```

Reply:
Done